### PR TITLE
PR#1: Setup project

### PR DESCRIPTION
**Pull request by lordksix**
In this PR, the changes are as follows:
- Create project
- Modify README.md
- Add SQL schema, data insert, and queries to SQL files.
- Add screenshots of queries.

Screenshots 
- Queries (find all animals whose name ends in "mon, list the name of all animals born between 2016 and 2019, list the name of all animals that are neutered and have less than 3 escape attempts, list the date of birth of all animals named either "Agumon" or "Pikachu", list name and escape attempts of animals that weigh more than 10.5kg, find all animals that are neutered)

queries1-6

- Queries (find all animals not named Gabumon, find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg)

queries7-8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -244,48 +166,33 @@ Feel free to check the [issues page](../../issues/).
...
 
-  - [Answer_2]
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (omarsalem7) [inline README.md:195]: - [x] Kindly consider fixing the broken MIT license link and add your own license

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @lordksix 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights

- The animal's table was created properly ✔️
- Correct git-flow ✔️

## Required Changes ♻️

- [x] It will be better to attach the screenshots of the output of the query in the body of the pull request as stated in the requirements of the project
- [x] _Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (lordksix) [reply README.md:195]: Thank you for the feedback. Fixed.

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @lordksix , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Screenshots of query results added to PR description :+1: 
:heavy_check_mark: Descriptive Readme file and PR

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**